onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -4433,8 +4433,8 @@ def _aten_native_batch_norm_inference_onnx(
         training_mode=training,
     )
     # Cannot return 2 dup output, so have to do twice with different variable name
-    empty_mean = op.Cast(op.Shape(input, 0, 0), to=FLOAT.dtype)
-    empty_var = op.Cast(op.Shape(input, 0, 0), to=FLOAT.dtype)
+    empty_mean = op.Cast(op.Shape(input, start=0, end=0), to=FLOAT.dtype)
```

Comment:
This line of code looks odd, wonder what it is trying to do! (I realize this has nothing to do with current PR.)

Reply:
Filed: https://github.com/microsoft/onnxscript/issues/786